lib/libUPnP/Neptune/Source/Core/NptConfig.h

```diff
@@ -61,7 +61,6 @@
 #define NPT_CONFIG_HAVE_SETENV
 #define NPT_CONFIG_HAVE_UNSETENV
 #if defined(TARGET_WINDOWS_STORE)
-#undef NPT_CONFIG_HAVE_GETENV
```

Comment:
this will break certification for uwp. see https://github.com/xbmc/xbmc/blob/master/lib/libUPnP/Neptune/Source/System/StdC/NptStdcEnvironment.cpp#L31. `getenv` is undefined on uwp. we may use `GetEnvironmentVariable` instead

Reply:
`GetEnvironmentVariable` was added with 10.0.16299